documents-d-archeologie-francaise.csl

```diff
@@ -424,214 +406,164 @@
   </citation>
   <bibliography>
     <sort>
-      <key macro="responsability"/>
+      <key macro="author-short"/>
       <key macro="year-date"/>
       <key macro="title"/>
     </sort>
     <layout>
-      <group prefix="" suffix="" delimiter=" ">
+      <group prefix="" suffix="&#9;" delimiter=" ">
```

Comment:
Could you try to use `display="block"` on the first `<group>` as well, and check whether that fixes things?